### ISSUE TITLE
File save dialog does not get focus

### ISSUE THREAD

**Issue by umpirsky** [atom/atom#15421]
### Description

May be related to https://github.com/atom/atom/issues/14139 and https://github.com/atom/atom/issues/14325.

### Steps to Reproduce

1. Open any text file.
2. Press `Ctrl + Shift + S`

**Expected behavior:** To have focus on file name input. No way to type name of the file without touching the mouse and focusing input, dialog is not focused. Only way around it is to press Ctrl + ` to gain focus.

**Actual behavior:** Dialog does not get focus.

**Reproduces how often:** [What percentage of the time does it reproduce?]

### Versions

```
atom --version
Atom    : 1.19.1
Electron: 1.6.9
Chrome  : 56.0.2924.87
Node    : 7.4.0

cat /etc/issue
Ubuntu 16.04.3 LTS
```

**Comment by rsese** [comment]
I can reproduce with 1.19.2 on Ubuntu 16.04 though I did notice that if I start Atom, the very first time I go through the steps to reproduce, the dialog and file name field has focus.  If I close the dialog, the next time I `ctrl-shift-s`, the editor has focus (i.e. anything I type is added to whatever the current buffer was).

I also tried with 1.20.0-beta2 and see the same behavior.

**Comment by rsese** [comment]
Also tagged as `linux` since I don't see this behavior on Windows or macOS.

**Comment by envygeeks** [comment]
Yeah, it's pretty annoying, and makes Atom way less efficient.  I don't like having to move my hand to my mouse just to save a file...

**Comment by umpirsky** [comment]
> I don't like having to move my hand to my mouse just to save a file

@envygeeks If that makes you less miserable, you can regain dialog focus with `Alt + ``.

**Comment by envygeeks** [comment]
It doesn't, because that's still more work than I should have had to of done.  Instead I just adjusted the Window rules to fix the problem outside of Atom since they won't fix the problem in Atom.  Luckily on Linux we can control our window policies.

**Comment by umpirsky** [comment]
@envygeeks How?

**Comment by envygeeks** [comment]
On Ubuntu 16.04 CCSM.  On GTK Mutter, on Ubuntu 17.10 this problem didn't exist but since GTK Mutter doesn't support fractal scaling that's proper I stuck with 16.04.

**Comment by ungb** [comment]
I can repro this off Atom latest (atom 1.23.3).

**Comment by lee-dohm** [comment]
Can anyone still reproduce this?
